src/Helper/HelperInterface.php

```diff
@@ -18,7 +18,7 @@ public function setView(Renderer $view);
     /**
      * Get the View object
      *
-     * @return Renderer
+     * @return Renderer|null
```

Comment:
All implementors tend to extend from AbstractHelper where the return type is plausibly null